src/components/checkbox/checkbox.tsx

```diff
@@ -0,0 +1,48 @@
+import React = require('react');
+
+interface CheckBoxProps {
+    text?: string;
+    value?: boolean;
+    boxIcon?: string;
+    tickIcon?: string;
+    onChange?: (value: boolean ) => any;
+
+}
+
+export class CheckBox extends React.Component<CheckBoxProps, {}> {
```

Comment:
we believe the only case we have to do it is we have to use `this.state`, otherwise it should be SFC